docs/interviewing_options.md

```diff
@@ -1 +1,134 @@
+solution within a specific timeframe, and normally think aloud their problem.
+
+Typically the problems candidates are given are algorithmic or puzzle type
+questions typically of what you mind find on [leetcode](https://leetcode.com) or [hackerrank](https://www.hackerrank.com/)
+
+#### Pros
+
+- identifying candidates whom are exceptionally brilliant, quick thinkers, and
+  extremely knowledgable about the specific technical topic at hand
+- See how well candidates can perform under pressure and scrutiny in real time.
+- If done online can help companies and candidates optimize on costs and time
+  of being on-site and in-person
+- Generally can be a low effort (cheap) way to screen out candidates whom might
+  not be technical enough for the role at hand. This can be effect in a company
+  where there are many many applicants.
+
+#### Cons
+
+- Can lead to a lot of false negatives
+- Can bias against people whom have family commitments (caregivers,
+  single parents, etc.)
+- Unintentionally discriminate based on age (favouring those whom might have
+  time to study, or have recently graduated from academia)
+- Creates a pressure cooker situation where candidates make mistakes[1]
+  - People whom might suffer from anxiety might perform well in these situations
+- Many of the questions asked in these formats are not real world type problems
+  and ultimately someone whom can invert a binary tree may not be able to
+  deliver on actual business problems faced by businesses
+- If candidate has potentially memorized the solution (from another interview
+  or seen it posted online) this again may not be indicative of their true
+  knowledge.
+- Unfair power imbalance between candidates and interviewers
+
+
+
+## Take-home tests (Homework, Challenges)
+
+Take-home tests are where a candidate is given an assignment to do on their own
+time and typically expected to return their completed assignment back in a set
+period time. this can be hours or days. Sometimes the results of the assignment
+become the topic of conversation during the on-site portion of the interview.
+
+#### Pros
+
+- Allows the candidate to work on their own time, space, environment setup, etc.
+- Candidate can use Google, and other online (legit) resources like they might
+  at a typical job
+- Can be done during non work hours allowing candidates more freedom
+- Less anxiety involved for candidate as no one is watching them
+- Certain types of tests might be conducive to automated testing to check
+  results and validity
+- Again this can be a cheap way to screen candidates as they don't need to come
+  on-site, and act as a stopper for those who are not super serious about
+  actually interviewing
+- Removes the power imbalance that can occur in whiteboard style interview.
+
+#### Cons
+
+- Possibility of candidates plagiarizing work
+- Can bias against people whom have family commitments (caregivers,
+  single parents, etc.) depending on the level of effort needed
```

Comment:
`Can bias against people who have caregiver commitments (parents, those taking care of a loved one, etc.) depending on the level of effort needed`

Reply:
👍